File: apps/android/app/src/main/java/com/rsr41/oracle/domain/model/Gender.kt
package com.rsr41.oracle.domain.model

enum class Gender(val apiValue: String, val displayName: String) {
    MALE("male", "남성"),
    FEMALE("female", "여성");

    companion object {
        fun fromApiValue(value: String): Gender = entries.find { it.apiValue == value } ?: MALE
    }
}

File: apps/ios/OracleIOS/OracleIOS/OracleIOS_Sources/Domain/Entities/Gender.swift
import Foundation

/// 성별 열거형
/// Android: domain/model/Gender.kt (lines 3-10)
enum Gender: String, Codable, CaseIterable, Identifiable {
    case male = "male"
    case female = "female"
    
    var id: String { rawValue }
    
    var displayName: String {
        switch self {
        case .male: return String(localized: "common.male")
        case .female: return String(localized: "common.female")
        }
    }
}
